### ISSUE TITLE
Cannot reproduce mAP with potholes data

### ISSUE THREAD

**Issue by tylertroy** [wangermeng2021/ScaledYOLOv4-tensorflow2#20]
I ran the command as specified in the docs to train on an RTX3090 using the potholes dataset as follows.

```bash
python train.py --use-pretrain True --model-type p5 --dataset-type voc --dataset dataset/pothole_voc --num-classes 1 --class-names pothole.names --voc-train-set dataset_1,train --voc-val-set dataset_1,val  --epochs 200 --batch-size 4 --multi-scale 320,352,384,416,448,480,512 --augment ssd_random_crop
```
However I cannot reproduce the mAP as presented.
![image](https://user-images.githubusercontent.com/17804750/158663498-f94f08d2-16b8-4f4a-a148-46addd79c9b3.png)

Similarly the training loss looks quite different.
![image](https://user-images.githubusercontent.com/17804750/158663627-caa226ac-38ee-409c-909c-8d658dbfec69.png)

Any thoughts on what I might be doing wrong? Or perhaps there has been a material change to the repo since those results were posted.

Feedback appreciated and thanks for providing this good work to the community.
